example/lib/page_remote_config_legacy.dart

```diff
@@ -120,14 +120,23 @@ class RemoteConfigPageLegacy extends StatelessWidget {
         child: Center(
             child: Column(
           children: [
+            // ignore: deprecated_member_use_from_same_package
```

Comment:
Either this or remove the @deprecated annotations above (if possible). This page already advertised as legacy. Thoughts?

Reply:
Yes, it is legacy, that is were ignoring deprecated warning

Reply:
yeap lets add it back kindly thank you